docs/dev_guides/changing_splink/contributing_to_docs.md

```diff
@@ -0,0 +1,58 @@
+## Building docs locally
+
+Before building the docs locally, you will need to follow the [development quickstart](./development_quickstart.md) to set up the necessary environment. You cannot skip this step, because some Splink docs Markdown is auto-generated using the Splink development environment.
+
+Once you've done that,
+to rapidly build the documentation and immediately see changes you've made you can use [this script](https://github.com/moj-analytical-services/splink/blob/master/scripts/make_docs_locally.sh)
+**outside your Poetry virtual environment**:
+
+```sh
+source scripts/make_docs_locally.sh
+```
+
+This is much faster than waiting for GitHub actions to run if you're trying to make fiddly changes to formatting etc.
+
+Once you've finished updating Splink documentation we ask that you run our spellchecker. Instructions on how to do this are given below.
+
+## Spellchecking docs
+
+When updating Splink documentation, we ask that you run our spellchecker before submitting a pull request. This is to help ensure quality and consistency across the documentation. Please note, the spellchecker _only works on markdown files_.
+
+To run the spellchecker on either a single markdown file or folder of markdown files, you can use the following script:
+
+```sh
+source scripts/pyspelling/spellchecker.sh <path_to_file_or_folder>
+```
```

Comment:
Should we also note that this only works on mac at present and further instructions for other operating systems will be released later?

Reply:
Good point, will update